ush/create_model_configure_file.py

```diff
@@ -24,7 +24,7 @@
 
 
 def create_model_configure_file(
-    cdate, cycle_type, cycle_subtype, run_dir, fhrot, nthreads, restart_hrs
+    cdate, cycle_type, cycle_subtype, stoch, run_dir, fhrot, nthreads, restart_hrs
 ):
```

Comment:
what is "stoch" for?

Reply:
The "stoch" item is something to let it know if it is an ensemble forecast or not (at least is the intent).